packages/client/components/promptResponse/PromptResponseEditor.tsx

```diff
@@ -102,14 +109,15 @@ const StyledEditor = styled('div')`
 
 interface Props {
   autoFocus?: boolean
+  teamId?: string
```

Comment:
+1 is there any case where team id would not be defined?

Reply:
Not necessarily, but this is only required for editing, so it seems wrong to make it required for cases when it's read-only (e.g. in the discussion drawer or summary)